galaxy.yml

```diff
@@ -1,17 +1,22 @@
-namespace: "@NAMESPACE@"
-name: "@NAME@"
+---
+namespace: "ovirt"
+name: "ovirt"
 version: 3.2.1
 authors:
-    - Martin Necas <mnecas@redhat.com>
+  - oVirt Developers <devel@ovirt.org>
 license:
-    - Apache-2.0
-    - GPL-3.0-or-later
+  - Apache-2.0
+  - GPL-3.0-or-later
 tags:
-    - collection
-    - infrastructure
-    - linux
-    - cloud
-    - virtualization
+  - collection
+  - infrastructure
+  - linux
+  - cloud
+  - virtualization
+dependencies:
+  "ansible.netcommon": "<3.0.0"
+  "ansible.posix": "*"
+  "ansible.utils": "<2.6.0"
```

Comment:
Does this collection really depend on `ansible.netcommon<3.0.0` and `ansible.utils<2.6.0`? Those versions are some three years old...

The current Ansible Community Package includes 7.2.0 and 5.1.2, respectively. I see a problem there. And it looks like Ansible 12 will even include ansible.netcommon 8.0.0 or later.

Reply:
This is a violation of the Ansible community package collection requirements: https://docs.ansible.com/ansible/devel/community/collection_contributors/collection_requirements.html#collection-dependencies

Never use `<` unless you have a **very** good reason for it (especially if it's not for a major release), and you make sure you quickly update it to new releases once they're out.

Reply:
Thanks for the useful comment.
Seems to be added here in the past: https://github.com/oVirt/ovirt-ansible-collection/pull/556

We will have a look and remove that versioned dependency.